docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/migrate-rancher-to-new-cluster.md

```diff
@@ -36,21 +37,22 @@ Install the [rancher-backup chart](https://github.com/rancher/backup-restore-ope
      helm repo update
      ```
 
-  1. Select and set `CHART_VERSION` variable with a 2.x.x rancher-backup release version:
+  1. Set a `CHART_VERSION` variable, selecting a `rancher-backup` chart version compatible with your version of Rancher. See the [support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions), within the **Rancher Apps / Cluster Tools** section, to see which `rancher-backup` versions are supported:
+
      ```bash
-     helm search repo --versions rancher-charts/rancher-backup
-     CHART_VERSION=<2.x.x>
+     CHART_VERSION=<x.x.x>
```

Comment:
```suggestion
     CHART_VERSION=<chart-version>
```

I think a text description as the placeholder is better here since it covers both version formats (e.g. `103.0.0+up4.0.0` vs `2.0.0`) whereas as `x.x.x` is confusing when the `up` format is expected for some Rancher versions.